plugin/db/driver.go

```diff
@@ -117,14 +117,28 @@ type Table struct {
 	IndexList []Index
 }
 
+// InstanceMetadata is the metadata for an instance.
+type InstanceMetadata struct {
```

Comment:
InstanceMeta to be consistent with other places.

Reply:
Done.